### PR TITLE
Bump version to 11

### PR DESCRIPTION
**Pull request by xoloki**

### CODE REVIEW

File: src/state_machine/coordinator/mod.rs [hunk 1]
```diff
@@ -972,4 +973,71 @@ pub mod test {
...
+            } else {
+                panic!("Message should have been SignatureShareResponse");
+            };
+            let share2 = if let Message::SignatureShareResponse(response) = message2 {
+                response.signature_shares[0].clone()
+            } else {
+                panic!("Message should have been SignatureShareResponse");
+            };
+
+            assert!(share1.z_i != share2.z_i);
```
Comment (djordon) [inline src/state_machine/coordinator/mod.rs]: I think the standard thing to do here is to use the `assert_ne!` macro for this.

File: src/state_machine/coordinator/mod.rs [hunk 2]
```diff
@@ -972,4 +973,71 @@ pub mod test {
 
         assert_eq!(signers, loaded_signers);
     }
+
+    pub fn gen_nonces<Coordinator: CoordinatorTrait, SignerType: SignerTrait>(
+        num_signers: u32,
+        keys_per_signer: u32,
+    ) {
```
Comment (djordon) [inline src/state_machine/coordinator/mod.rs]: I know you now what this test does, but I think that we should add a bit more documentation to this test. It's not clear what is being tested an why the outcome is the correct one. I mean, I know what's going on but someone who is being onboarded here might have a tough time figuring a lot of this out. So I'd add more detailed comments during the test and add a description above the test function as well.

**Review by djordon — APPROVED** [review]
This looks good.

I left a comment about adding documentation/comments in this test. If you agree, please add them.

File: src/state_machine/coordinator/mod.rs (hunk 2 above)
Comment (xoloki) [reply src/state_machine/coordinator/mod.rs]: Done

**Pull request merged**